Gems/ROS2/Code/Source/RobotImporter/Utils/SourceAssetsStorage.cpp

```diff
@@ -255,6 +256,7 @@ namespace ROS2::Utils
         const AZStd::string& urdfFilename,
         const AZStd::unordered_set<AZStd::string>& colliders,
         const AZStd::unordered_set<AZStd::string>& visuals,
+        const AZStd::string& outputDirSuffix,
```

Comment:
This parameter can be an `AZStd::string_view` which is more flexible than an `const AZStd::string&` parameter and avoids temporary heap allocations when passing in a string literal.